Polyhedron/demo/Polyhedron/Plugins/Three_examples/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ find_package(Qt5
              COMPONENTS            OpenGL Script Svg Xml
              OPTIONAL_COMPONENTS   ScriptTools WebSockets)
 
-if(CGAL_TEST_SUITE)
+if(RUNNING_CGAL_AUTO_TEST)
```

Comment:
Be careful with the wording:
- `RUNNING_CGAL_AUTO_TEST` is the name of the CMake *variable*.
- `CGAL_TEST_SUITE` is the name of the C++ preprocessor *macro*.

A *macro* in CMake language is similar to a function.